Content.Shared/Players/PlayTimeTracking/PlayTimeTrackingShared.cs

```diff
@@ -6,4 +6,9 @@ public static class PlayTimeTrackingShared
     /// The prototype ID of the play time tracker that represents overall playtime, i.e. not tied to any one role.
     /// </summary>
     public const string TrackerOverall = "Overall";
+
+    /// <summary>
+    /// The prototype ID of the play time tracker that represents admin time, when a player is in game as admin.
+    /// </summary>
+    public const string TrackerAdmin = "Admin";
```

Comment:
Use validateprototype.

Reply:
`[ValidatePrototypeId<PlayTimeTrackerPrototype>]
    public const string TrackerAdmin = "Admin";`
    
 Added this one in.